tests/controllers/ControllerFactory.ts

```diff
@@ -0,0 +1,15 @@
+import { Connection } from 'typeorm';
+import FeedbackService from '../../services/FeedbackService';
+import { FeedbackController } from '../../api/controllers/FeedbackController';
+
+export class ControllerFactory {
+  private static feedbackController: FeedbackController = null;
```

Comment:
`DatabaseConnection` has a class-wide connection because it makes managing connections simpler like with closing them out, but it's fine for `ControllerFactory` to instantiate the controller  every time.

Reply:
I feel like it's alright to leave the controllers as singletons, just so that service/controller objects don't have to be re-created again every time a controller is invoked, even though that overhead is fairly small.